wbb/modules/images.py

```diff
@@ -0,0 +1,63 @@
+from wbb.utils import cust_filter
+from wbb import app, WALL_API_KEY
+import urllib.request
+import json
+from random import randint
+import requests as r
+
+__MODULE__ = "Images"
+__HELP__ = '''/cat - Get Cute Cats Images
+/dog - Get Cute Dogs Images
+'''
+
+
+@app.on_message(cust_filter.command(commands=("cat")))
+async def cat(client, message):
+    with urllib.request.urlopen(
+        "https://api.thecatapi.com/v1/images/search"
+    ) as url:
+        data = json.loads(url.read().decode())
+    cat_url = (data[0]['url'])
+    await message.reply_photo(cat_url)
+
+
+@app.on_message(cust_filter.command(commands=("dog")))
+async def dog(client, message):
+    with urllib.request.urlopen(
+        "https://api.thedogapi.com/v1/images/search"
+    ) as url:
+        data = json.loads(url.read().decode())
+    dog_url = (data[0]['url'])
+    await message.reply_photo(dog_url)
+
+
+@app.on_message(cust_filter.command(commands=("wall")))
+async def wall(client, message):
+    app.set_parse_mode("markdown")
+    m = await message.reply_text("Searching!")
+    try:
+        term = (message.text.split(None, 1)[1])
+    except IndexError:
+        await m.edit("```\"/wall\" needs a keyword argument```")
+
```

Comment:
just ```"`\"/wall\" needs a keyword argument`"``` would be enough, three backticks is for multiple lines

Reply:
Got it